injected/integration-test/fingerprint.spec.js

```diff
@@ -5,6 +5,7 @@ import { test as base, expect } from '@playwright/test'
 import { testContextForExtension } from './helpers/harness.js'
 import { createRequire } from 'node:module'
 
+// eslint-disable-next-line no-redeclare
```

Comment:
Are we importing 'require' as a global, we shouldn't

Reply:
The new config [assumes NodeJS globals in test files](https://github.com/duckduckgo/content-scope-scripts/pull/1185/files#diff-a32a0887ed9d1d707bbb3b845b7df7fd40e673c47e7b60a3ebd896b68d3b8839R69-R76)